modules/develop/pages/intro.adoc

```diff
@@ -73,4 +73,11 @@ It enables you to execute code either in response to mutations, or as scheduled
 
 The Analytics Service is a parallel data management capability for Couchbase Server, designed to run ad-hoc, analytical queries efficiently over many records.
 
-* xref:analytics:introduction.adoc[Introduction to Analytics]
\ No newline at end of file
+* xref:analytics:introduction.adoc[Introduction to Analytics]
+
+== Integrations
+
+Integrations that Couchbase or partners have developed to enhance Couchbase, Capella, and partner services --
+a vital tool for many larger development projects.
+
+* xref:server:develop:integrations.adoc[]
```

Comment:
You shouldn't need to include the `server` component in the xref coordinates, as this page is already in a "server" branch of `docs-devex` &mdash; the Antora component identifier will ensure that the playbook adds this page in the `server` component.

If you include the `server` component in these coordinates, you'll have to remember to change it to `cloud` when you port this change to the "cloud" branches of `docs-devex`. If you leave out the `server` component, you can port this change to the "cloud" branches without having to change the xref co-ordinates. Generally I try to stop the "cloud" and "server" branches diverging unnecessarily.

(Test the build to make sure the xref still works, of course.)

```suggestion
* xref:develop:integrations.adoc[]
```

Reply:
This comment above assumes that `integrations.adoc` is going to be in the docs-devex repo of course.